styles.css

```diff
@@ -268,3 +277,225 @@ textarea {
 }
 
 /* footer end */
+
+/* Desktop Version Start */
+
+@media (min-width: 1440px) {
+  /* desktop Nav */
+  body {
+    background-color: #e5e5e5;
+  }
+
+  .mobile-view {
+    width: auto;
+    background-color: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  nav h2,
+  nav img {
+    display: none;
+  }
+
+  nav {
+    display: block;
+  }
+
+  .destop-menu {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    list-style: none;
+    padding: 20px 0;
+    margin: 0;
+  }
+
+  .destop-menu-list {
+    margin: 0 30px;
+    font-size: 1.2em;
+  }
+
+  /* Desktop Nav End */
+
+  /* Desktop Header Start */
+  header {
+    background-image: none;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    gap: 24px;
+    position: absolute;
+    width: 920px;
+    height: 756px;
+    left: 254px;
+    top: 98px;
+  }
+
```

Comment:
You have correctly implemented the background image of the headline section. Kindly Adjust the margins to match the design. Consider the distance between the last point of the image and the "My Recent Works" heading.

Your design:
![image](https://user-images.githubusercontent.com/20554583/199488579-5df33723-fc5e-4877-8f1b-570e70e79cb2.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/199488308-6eaff98e-bc56-4348-9877-1960601ee354.png)